client/state/embeds/test/reducer.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * External dependencies
+ */
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import reducer, { siteItems, urlItems } from '../reducer';
+import { EMBED_RECEIVE, EMBEDS_RECEIVE } from 'state/action-types';
+
+describe( 'reducer', () => {
+	const siteId = 12345678;
+
+	test( 'should export expected reducer keys', () => {
+		const state = reducer( undefined, {} );
+		expect( state ).toMatchSnapshot();
+	} );
+
+	describe( 'siteItems', () => {
+		const sourceEmbeds = [ 'something' ];
+		const resultEmbeds = [ /something/ ];
```

Comment:
Since we're testing the `normalizeEmbeds` utility function separately, do we really want to care about the data normalization here, too?

It looks a bit like an integration test to me. In case we wanted to update the data normalization rules, we would have to update both files which seems unnecessary.

What do you think?

Reply:
Totally valid point 👍 Mocked the normalization utility in the reducer tests in https://github.com/Automattic/wp-calypso/pull/38608/commits/fb9fb59a1b24d8e00215ae25b5c5699f307bfed6.